crates/oxc_semantic/src/jsdoc/builder.rs

```diff
@@ -156,7 +156,8 @@ impl<'a> JSDocBuilder<'a> {
         }
 
         // Remove the very first `*`
-        Some(JSDoc::new(&comment_content[1..]))
+        let jsdoc_span = Span::new(comment_span.start + 1, comment_span.end);
+        Some(JSDoc::new(&comment_content[1..], jsdoc_span))
```

Comment:
pass in `comment_span` and compute `jsdoc_span` inside `JSDoc::new` instead? Feels weird to pass in a `jsdoc_span`.

Reply:
Since I was passing a string slice with the `*` removed to `JSDoc::new()`, I was also passing the `Span` with the `*` removed.

Do you mean both should be done on the `JSDoc` side? 👀